scripts/include/common.sh

```diff
@@ -14,15 +14,19 @@ export BUILD_DIR=build${CLUSTER_NAME}
 
 export ROOT_DIR="$(git rev-parse --show-toplevel)"
 export CHART_URL="${CHART_URL:-}"
-export KUBECONFIG=$PWD/kubeconfig
+export KUBECONFIG=$BUILD_DIR/kubeconfig
```

Comment:
Mmm in this way it won't be anymore an absolute path, right? but afaict shouldn't be an issue (the only thing is it would allow `kubectl` to be called inside the make targets also if there are any pushd/popd around)

Reply:
nvm, didn't saw the commit rightafter